examples/wannier90.jl

```diff
@@ -0,0 +1,64 @@
+# # DFTK - Wannier90 interface
+#
+# DFTK features an interface with the program
+# [Wannier90](http://www.wannier.org/),
+# in order to compute maximally-localized Wannier functions (MLWFs)
+# from an initial self consistent field calculation.
+# All processes are handled by calling the routine `run_wannier90`.
+#
+# This example shows how to obtain the MLWFs corresponding
+# to the first eight bands of silicon. Since the bands 5 to 8 are entangled,
+# 12 bands are first computed to obtain 8 MLWFs by a disantanglement procedure.
+# We first perfom a SCF calculation.
+
+using DFTK
+
+a = 10.26
+lattice = a / 2*[[-1.  0. -1.];
+                 [ 0   1.  1.];
+                 [ 1   1.  0.]]
+
+Si = ElementPsp(:Si, psp=load_psp("hgh/pbe/Si-q4"))
+atoms = [ Si => [zeros(3), 0.25*[-1,3,-1]] ]
```

Comment:
This is not the same cell as the other silicon examples right? Is this because you want to follow a tutorial in wannier90? If yes say explicitly and link to the corresponding tutorial

Reply:
Nope I have to admit I don't know why this cell is different ! I might have done a copy-paste a little too fast. I will take the one from example/silicon.jl :+1: